helm_deploy/hmpps-education-and-work-plan-api/values.yaml

```diff
@@ -42,6 +42,9 @@ generic-service:
       DB_NAME: "database_name"
       DB_USER: "database_username"
       DB_PASS: "database_password"
+    digital-prison-reporting:
```

Comment:
Have you created this secret yet in `kubectl` for the 3 namespaces?
If not we'll have to hold off merging this as it wont deploy (k8s will moan bitterly that we are referencing a secret to pass into the service pod that doesn't exist)

Reply:
Yep – added to all three namespaces.